### ISSUE TITLE
README: Installation process

### ISSUE THREAD

**Issue by smalderez** [transcend-io/terragrunt-atlantis-config#341]
## Bug report

it seems the README is not updated
"wget https://github.com/transcend-io/terragrunt-atlantis-config/releases/download/v${var.terragrunt_atlantis_config_version}/terragrunt-atlantis-config_${var.terragrunt_atlantis_config_version}_linux_amd64.tar.gz" does not work anymore.

With latest version, "wget https://github.com/transcend-io/terragrunt-atlantis-config/releases/download/v1.18.0/terragrunt-atlantis-config_1.18.0_linux_amd64.tar.gz" return a 404.

**Comment by carnei-ro** [comment]
yeah, it is not `tar.gz` anymore, but at least now we have linux `arm64` 🎉

**Comment by robertomoutinho** [comment]
Should we have the suffix back ? Feels like having a file without its extension is not optimal.

**Comment by relu** [comment]
I'm not a maintainer so I don't really have a say, though I can provide my view on it.

My preference is usually to not compress binaries for GitHub release artifacts because I feel it just adds unnecessary steps to installation scrips.
Pleae note that the new files are binaries and not compressed archives missing the extension.

I assume the problem here is that old installation scripts are failing due to the change in the file type which broke the previous convention. It would have been good if a note was added to the release mentioning this breaking convention change and the update to the readme was missed, which is my bad.
